pkg/asset/installconfig/platformcredscheck.go

```diff
@@ -3,6 +3,7 @@ package installconfig
 import (
 	"context"
 	"fmt"
+	"github.com/openshift/installer/pkg/types/ovirt"
```

Comment:
this should be moved into the last group block of imports

Reply:
done